### PR TITLE
feat(i18n): Add Hungarian translation (hu-HU)

### PR DESCRIPTION
**Pull request by iamdtms**
cc: https://github.com/userquin

### CODE REVIEW

**Comment by userquin** [comment]
@iamdtms to ping someone in GH, `/cc @xxxxx`

**Comment by userquin** [comment]
@iamdtms I'll try to fix json indentation later

**Comment by userquin** [comment]
There are a few english entries in the json file, you should remove them or finish the translation: there is a page for missing entries per country/locale in the docs.

If we keep translation entries in english there is no way to know the status, so, can you review it again?

File: config/i18n.ts [hunk 1]
```diff
@@ -72,6 +72,10 @@ export const countryLocaleVariants: Record<string, (LocaleObjectData & { country
     { country: true, code: 'pt-PT', name: 'Português (Portugal)' },
     { code: 'pt-BR', name: 'Português (Brasil)' },
   ],
+  hu: [
+    { country: true, code: 'hu-HU', name: 'Hungarian (Hungarian)' },
+    { code: 'hu-HU', name: 'Hungarian (Hungarian)' },
```
[the remark this answers is not in the document]
Comment (iamdtms) [reply config/i18n.ts]: Surely not, will be fixed in the next PR.

**Comment by iamdtms** [comment]
Please  help, it is the Hungarian translation at ~15%. 
Config issue fixed.

**Comment by userquin** [comment]
@iamdtms I cannot help, I have no idea about Hungarian, if it is about the error in CI just ignore it, we'll fix it when translation finished.

Are you on discord server (Elk server)?

**Comment by iamdtms** [comment]
Sure, getting back when texting is 100%.

**Comment by iamdtms** [comment]
3/7 ready

**Comment by iamdtms** [comment]
v0.1 release: 
https://github.com/iamdtms/elk-hungarian/releases/tag/v0.1
cc: @userquin

**Comment by userquin** [comment]
Can you review this entry `Language` should be capitalized `nyelv` => `Nyelv`?

imagen

File: locales/hu-HU.json [hunk 2]
```diff
@@ -0,0 +1,695 @@
...
+      "default": " (default)",
+      "font_size": "betű mérete",
+      "label": "Felület",
+      "light_mode": "Világos",
+      "system_mode": "Rendszer",
+      "theme_color": "téma színe"
+    },
+    "language": {
+      "display_language": "Megjelenítés nyelve",
+      "label": "nyelv",
```
Comment (userquin) [inline locales/hu-HU.json:430]: this one

Comment (iamdtms) [reply locales/hu-HU.json]: fixed

**Review by userquin — APPROVED** [review]

**Comment by iamdtms** [comment]
personal tag & release removed

**Comment by iamdtms** [comment]
merge elk:main success
cc: @patak-dev

**Comment by iamdtms** [comment]
Would you please review this, @patak-dev?

**Comment by iamdtms** [comment]
All checks have passed, merging is still blocked like last week. PR updated several times with merging branch 'elk-zone:main' into local:main. This translation pack is important to me.

**Review by patak-dev — APPROVED** [review]
Köszönöm a fordítást! OSS projektekben normális, hogy a PR-ok összeolvadása eltart egy ideig 🙏🏼

**Pull request merged**